src/workerd/server/tests/python/sdk/worker.py

```diff
@@ -51,8 +51,11 @@ async def on_fetch(request):
     elif request.url.endswith("/undefined_opts"):
         # This tests two things:
         #   * `Response.redirect` static method
-        #   * that other options can be passed into `fetch`
-        resp = await fetch("https://example.com/redirect", redirect="manual")
+        #   * that other options can be passed into `fetch` (so that we can support
+        #       new options without updating this code)
+        resp = await fetch(
+            "https://example.com/redirect", redirect="manual", foobarbaz=42
+        )
```

Comment:
I guess we could test that we receive `foobarbaz` on the other side by mocking the JS fetch. In this case, we just test that we don't crash with this option but we could discard it without failing the test?